src/app/shared/form/builder/ds-dynamic-form-ui/models/relation-group/dynamic-relation-group.component.html

```diff
@@ -1,14 +1,14 @@
 <a *ngIf="!(formCollapsed | async)"
    class="close position-relative"
-   ngbTooltip="{{'form.group-collapse-help' | translate}}"
+   ngbTooltip="{{'Collapse-help' | translate}}"
    placement="left">
   <span class="fas fa-angle-up fa-fw fa-2x"
         aria-hidden="true"
         (click)="collapseForm()"></span>
 </a>
 <a *ngIf="(formCollapsed | async)"
    class="close position-relative"
-   ngbTooltip="{{'form.group-expand-help' | translate}}"
+   ngbTooltip="{{'Expand-help' | translate}}"
```

Comment:
This should be `{{'Click here to expand and add more elements' | translate}}`.

Reply:
thanks, fixed today